src/components/icons/workflow.js

```diff
@@ -26,10 +26,7 @@ const WorkflowIcon = ({ className }) => (
       />
     </mask>
     <g mask="url(#mask0_3908_33753)">
-      <path
-        d="M22.4807 12.7497V11.2498H17.0115L14.7884 17.0228L9.21149 2.80371L5.91536 11.2498H1.49996V12.7497H6.96921L9.21151 6.96709L14.798 21.1862L18.0653 12.7497H22.4807Z"
-        fill="#D9DCDE"
-      />
+      <path d="M22.4807 12.7497V11.2498H17.0115L14.7884 17.0228L9.21149 2.80371L5.91536 11.2498H1.49996V12.7497H6.96921L9.21151 6.96709L14.798 21.1862L18.0653 12.7497H22.4807Z" />
```

Comment:
You can simplify this SVG file since it doesn’t actually need a mask or other things to be wrapped it. Including those elements just makes it harder to apply styling with CSS.

This can just be something like this 

```
const WorkflowIcon = ({ className }) => (
  <svg className={className} viewBox="0 0 24 24">
    <path d="M22.4807 12.7497V11.2498H17.0115L14.7884 17.0228L9.21149 2.80371L5.91536 11.2498H1.49996V12.7497H6.96921L9.21151 6.96709L14.798 21.1862L18.0653 12.7497H22.4807Z" />
  </svg>
);
```